crates/ra_hir_def/src/nameres/raw.rs

```diff
@@ -145,14 +145,21 @@ impl_arena_id!(Import);
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct ImportData {
     pub(super) path: ModPath,
-    pub(super) alias: Option<Name>,
+    pub(super) alias: ImportAlias,
     pub(super) is_glob: bool,
     pub(super) is_prelude: bool,
     pub(super) is_extern_crate: bool,
     pub(super) is_macro_use: bool,
     pub(super) visibility: RawVisibility,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ImportAlias {
+    NoAlias,
```

Comment:
Could you please put comments near `NoAlias` and `Alias(Name)` like you did with `Unnamed`? I imagine I wouldn't be able to understand what `NoAlias` means without the context of this PR...

Reply:
Sure, makes total sense

Reply:
Can you make them doc comments for each variant? Then they will show up in the IDE.

Reply:
Maybe it could be called `OriginalName` or `Default` :thinking:

Reply:
I suggest

```
enum ImportAlias { Name(Name), Underscore }
```

and using `Option` for no-alias case